### ISSUE TITLE
checkpoint file so large(643M)

### ISSUE THREAD

**Issue by clhne** [luckycallor/InsightFace-tensorflow#5]
Hi, luckycallor
I find the checkpoint file so large, about 643M, are there any idea to reduce the size?

**Comment by luckycallor** [comment]
@clhne 
The currently released pretrained model is based on modified ResNet_v2-50 network, which has lots of weights. In the future we will release pretrained model based on smaller models such as mobile net and shuffle net, which will be suitable to mobile devices.

**Comment by ShiHangY** [comment]
Hi, luckycallor
I want to reduce the checkpoint file,by excluding classified layer
` 
sess = tf.Session()
embd_vars = tf.global_variables(scope='embd_extractor')
new_saver = tf.train.Saver(embd_vars)#embd_vars
sess.run(tf.global_variables_initializer())
new_saver.save(sess,'./model/best-m-123000')
`
But,the embding output is different from original.

**Comment by WildChlamydia** [comment]
@luckycallor Don't you have some pretrained bigger model? 101 or 152?

**Comment by rlrahulkanojia** [comment]
where to find checkpoint file?

**Comment by clhne** [comment]
> where to find checkpoint file?

You can find the ckpt file in the path: 
output_dir/checkpoints
output_dir is the path you set in configs/config_***.yaml file.

**Comment by rlrahulkanojia** [comment]
thanks. Is the checkpoint files of given pre-trained weights available?
